beets/mediafile.py

```diff
@@ -2005,6 +2007,44 @@ def update(self, dict):
         out_type=float,
     )
 
+    # EBU R128 fields.
+    r128_track_gain = MediaField(
+        MP3DescStorageStyle(
+            u'R128_TRACK_GAIN'
+        ),
+        MP3DescStorageStyle(
+            u'r128_track_gain'
+        ),
```

Comment:
Well, the RFC always refers to it in upper case, so I guess we should go with that for consistency.